lib/winston/transports/http.js

```diff
@@ -162,6 +177,14 @@ module.exports = class Http extends TransportStream {
     return stream;
   }
 
+  /**
+   * Flush the transport.
+   * @returns {Promise<void>} - A promise that completes when all active http calls are complete.
+   */
+  flush() {
+    return Promise.all(this.callsInProgress.values());
```

Comment:
This actually returns `Promise<void[]>` consider replacing with 
```javascript
return (async () => {
      await Promise.all(this.callsInProgress.values());
 })();
```

Reply:
good callout, i just changed the return type